Orange/tests/test_evaluation_testing.py

```diff
@@ -381,11 +373,10 @@ def test_probs(self):
         self.assertEqual(results.predicted.shape, (2, len(test)))
         np.testing.assert_equal(results.predicted, np.ones((2, 50)))
         probs = results.probabilities
-        self.assertTrue((probs[:, :, 0] == 0).all())
+        self.assertEqual((probs[:, :, 0]).all(), 0)
```

Comment:
Sorry to nitpick here, but please revert this assertion. It was way clearer before. :smiley: